client/ayon_core/tools/utils/folders_widget.py

```diff
@@ -192,7 +192,11 @@ def _on_refresh_thread(self, thread_id):
             or thread_id != self._current_refresh_thread.id
         ):
             return
-        folder_items, folder_type_items = thread.get_result()
+        if thread.failed:
+            # TODO visualize that refresh failed
+            folder_items, folder_type_items = {}, {}
```

Comment:
Can we do anything to reproduce this particular issue when the thread would fail?

Reply:
It may wary, could be because server failed, or like reason of this PR, missing key in data.